packages/server/src/employees/employees.controller.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
```

Comment:
You can probably leave this in now, but we probably wont be using a controller for the time tracker.